.changeset/slow-eggs-grab.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/repo-tools': minor
+'@backstage/plugin-proxy-backend': patch
+---
+
+Api reference documentation improvements
+
+- breadcrumbs links semantics as code spans
+- new `@config` annotation to describe related config keys
```

Comment:
Gotta split up this changeset, this message really only makes sense for `repo-tools` right?

Reply:
Yes @Rugvip, you are right this only applies to `repo-tools`.
how do I need to split it? creating a different changeset file for `proxy-backend` is enough?

Reply:
done,  let me know if looks ok now